.changeset/funny-owls-buy.md

```diff
@@ -0,0 +1,6 @@
+---
+'@directus/api': patch
+'@directus/app': patch
+---
+
+Added multi-domain OAuth/OpenID support
```

Comment:
```suggestion
Added multi-domain SSO support
```

Reply:
Multi domain support was only restricted from Oauth/OpenID before this fix.